checkout/models.py

```diff
@@ -2,44 +2,34 @@
 from products.models import Product
 from django.contrib.auth.models import User
 
-class Order(models.Model):
-    full_name = models.CharField(max_length=50, blank=False)
-    phone_number = models.CharField(max_length=20, blank=False)
-    country = models.CharField(max_length=40, blank=False)
-    postcode = models.CharField(max_length=20, blank=True)
-    town_or_city = models.CharField(max_length=40, blank=False)
-    street_address1 = models.CharField(max_length=40, blank=False)
-    street_address2 = models.CharField(max_length=40, blank=False)
-    county = models.CharField(max_length=40, blank=False)
-    date = models.DateField()
-
-    def __str__(self):
-        return "{0}-{1}-{2}".format(self.id, self.date, self.full_name)
-
-
-class OrderLineItem(models.Model):
-    order = models.ForeignKey(Order, null=False)
-    product = models.ForeignKey(Product, null=False)
-    quantity = models.IntegerField(blank=False)
-
-    def __str__(self):
-        return "{0} {1} @ {2}".format(
-            self.quantity, self.product.name, self.product.price)
-
 
 class TimestampedModel(models.Model):
+    """
+    this model will post order to admin databse with order history
+    """
     class Meta : 
         abstract= True
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
 
+    
+
 
 class OrderHistory(TimestampedModel):
+    """
+    this model holds the user order history 
+    """
     user = models.ForeignKey(User, on_delete = models.CASCADE)
```

Comment:
Order should be linked to the Customer, not the User. Use the Customer everywhere on the app as far as you have a relationship between User and Customer.